manual/cops_style.md

```diff
@@ -1996,6 +1996,25 @@ at_exit { puts 'Goodbye!' }
 
 * [https://rubystyle.guide#no-END-blocks](https://rubystyle.guide#no-END-blocks)
 
+## Style/EndlessRange
```

Comment:
@koic I guess it won't be a bad idea to somehow incorporate the required Ruby version in this docs snippet. Might be a good idea to draw people's attention to the fact that some cops run only on certain Rubies.

Reply:
This idea looks good to me! So, each Ruby version supports new syntax and APIs. It can be a work independent of this PR :-)

Reply:
Yeah, totally. I just thought of this while going over this PR and thought I should share my thoughts with you.